package.json

```diff
@@ -4,11 +4,12 @@
   "license": "MIT",
   "private": true,
   "scripts": {
+    "bootstrap": "lerna bootstrap",
     "start": "lerna run start --parallel",
     "build": "lerna run build",
     "test": "lerna run test -- ",
     "lint": "lerna run lint",
-    "publish": "lerna publish --graph-type all --yes -m \"chore(release): publish [ci skip]\" --conventional-commits"
+    "publish-packages": "lerna publish --yes -m \"chore(release): publish [ci skip]\" --conventional-commits --no-git-reset --force-publish"
```

Comment:
renamed the action to rule that out as a factor - altho it shouldn't be anyway - done on the basis of VSCode having a description of the "publish" script as something run after publishing has done

the `--no-git-reset` and `--force-publish` tags re documented here: https://github.com/lerna/lerna/tree/master/commands/version
the reset i guess could potentially cause the build to get deleted.  the force-publish may help ensure our versions stay in sync